contracts/governance_admin/src/query/query_refundable_funds.rs

```diff
@@ -117,10 +117,45 @@ pub fn query_refundable_funds(
     })?;
 
     let (final_refundable_deposits, refund_reason) = match proposal_status {
-        ProposalStatus::Rejected => Ok((
-            user_total_deposits,
-            RefundReason::ProposalRejectedFullRefund,
-        )),
+        ProposalStatus::Rejected => {
+            // if rejected, check if it was vetoed or not
+            let tally_result = proposal.final_tally_result.unwrap();
+            let veto_no_count = Uint128::from_str(&tally_result.no_with_veto_count).unwrap();
+            let yes_count = Uint128::from_str(&tally_result.yes_count).unwrap();
+            let no_count = Uint128::from_str(&tally_result.no_count).unwrap();
+            let abstain_count = Uint128::from_str(&tally_result.abstain_count).unwrap();
+
+            let total_count = veto_no_count + yes_count + no_count + abstain_count;
+            let veto_percentage = Decimal256::from_ratio(veto_no_count, total_count);
```

Comment:
I think we shouldn't be doing total vote count, as there may a case where some validators (`< 1/3`) vetoed and others did not even vote or only few of them voted.
Can check if not voting counts as `abstain` when voting period ends, if not then we need to get total voting power to calculate `veto_percentage`.

Reply:
We have decided to not handle this case. Deposit is void for the proposal creator in case quorum is not met, and proposal is NoWithVeto.